### PR TITLE
PD-851 Update Netdata Tutorial to Cobia

### PR DESCRIPTION
**Pull request by micjohnson777**
This PR updates the InstallNetdataApp.md article with cobia instructions and screenshots, and new Netdata local dashboard and Nodes screenshots.

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/SCALE/SCALETutorials/Apps/CommunityApps/InstallNetdataApp.md [hunk 1]
```diff
@@ -15,97 +15,143 @@ tags:
...
+The SCALE Netdata app does not require advance preparation.
 
-Before using SCALE to install the Netdata application you need to configure TrueNAS SCALE storage for the Netdata application to use.
+You can allow SCALE to automatically create storage volumes for the Netdata app or you can create specific datasets to use for configuration, cache, and library storage and extra storage volumes in the container pod.
+If using specific datasets, create these before beginning the app installation.
 
-Verify the [local administrator]({{< relref "ManageLocalUsersSCALE.md" >}}) account has sudo permissions enabled.
+The administrator account must have sudo permissions enabled. 
+To verify, go to **Credentials > Local User**. 
+Click on the administrator user (e.g., admin), then click **Edit**. Scroll down to the sudo permissions.
```
Comment (DjP-iX) [inline content/SCALE/SCALETutorials/Apps/CommunityApps/InstallNetdataApp.md:31]: What should they check for or enable here? **Allow all sudo commands**, **Allow all sudo commands with no password**, both?

File: content/SCALE/SCALETutorials/Apps/CommunityApps/InstallNetdataApp.md [hunk 2]
```diff
@@ -15,97 +15,143 @@ tags:
...
+
+## Using the Netdata Web Portal
+After deploying the SCALE Netdata app click on **Web Portal** to open the Netdata agent local dashboard. 
+This dashboard provides a system overview with CPU usage and other vital statistics.
 
-2. The Netdata agent displays a limited portion of the reporting capabilities of the Netdata app. Click on the *Node View* tab to better understand the differences between the Netdata agent and Netdata Cloud. Evaluate your system reporting needs.
+{{< trueimage src="/images/SCALE/Apps/NetdataWebPortalLocalDashboard.png" alt="Netdata Web Portal Local Dashboard" id="Netdata Web Portal Local Dashboard" >}}
 
-   ![InstallNetDAppNetDAgentDashNoCloudSCALE](/images/SCALE/Apps/InstallNetDAppNetDAgentDashNoCloudSCALE.png "Netdata Agent Node View")
+The Netdata dashboard displays a limited portion of the reporting capabilities of the Netdata app.
```
Comment (DjP-iX) [inline content/SCALE/SCALETutorials/Apps/CommunityApps/InstallNetdataApp.md]: Not quite sure what you're saying here. Is it that the Netdata dashboard (accessed through SCALE Netdata app) is limited compared to the Netdata Cloud? There are four terms in 147-148 (Netdata dashboard, Netdata app--which may or may not be the SCALE app?--, Netdata agent, and Netdata Cloud) and I'm not sure which is referring to what, if there is an overlap between the first two terms and the second two, etc.

File: content/SCALE/SCALETutorials/Apps/CommunityApps/InstallNetdataApp.md [hunk 3]
```diff
@@ -15,97 +15,143 @@ tags:
...
+## Using the Netdata Web Portal
+After deploying the SCALE Netdata app click on **Web Portal** to open the Netdata agent local dashboard. 
+This dashboard provides a system overview with CPU usage and other vital statistics.
 
-2. The Netdata agent displays a limited portion of the reporting capabilities of the Netdata app. Click on the *Node View* tab to better understand the differences between the Netdata agent and Netdata Cloud. Evaluate your system reporting needs.
+{{< trueimage src="/images/SCALE/Apps/NetdataWebPortalLocalDashboard.png" alt="Netdata Web Portal Local Dashboard" id="Netdata Web Portal Local Dashboard" >}}
 
-   ![InstallNetDAppNetDAgentDashNoCloudSCALE](/images/SCALE/Apps/InstallNetDAppNetDAgentDashNoCloudSCALE.png "Netdata Agent Node View")
+The Netdata dashboard displays a limited portion of the reporting capabilities of the Netdata app. 
+Click on the **Nodes** tab to better understand the differences between the Netdata agent and Netdata Cloud. Evaluate your system reporting needs.
```
Comment (DjP-iX) [inline content/SCALE/SCALETutorials/Apps/CommunityApps/InstallNetdataApp.md:147]: see above

**Review by DjP-iX — CHANGES_REQUESTED** [review]

File: content/SCALE/SCALETutorials/Apps/CommunityApps/InstallNetdataApp.md (hunk 2 above)
Comment (micjohnson777) [reply content/SCALE/SCALETutorials/Apps/CommunityApps/InstallNetdataApp.md]: This entire section is talking about the Netdata dashboard accessed through their web portal as indicated by the level 2 header. I've added additional clarifying text.

**Review by DjP-iX — APPROVED** [review]

**Pull request merged**